### PR TITLE
add missing ws-tests files

### PR DESCRIPTION
**Pull request by jar398**
Not much here yet

### CODE REVIEW

**Comment by jar398** [comment]
http: should forward to https:.  I think the real problem is that it doesn't. Maybe some special option is needed for the requests library to make it follow the redirect.

**Comment by jimallman** [comment]
Here's [a bit from the requests documentation](http://docs.python-requests.org/en/latest/user/quickstart/#redirection-and-history) on redirection and history, and some [possibly related gotchas](https://github.com/kennethreitz/requests/issues/2618) regarding fetching large payloads.

**Comment by jar398** [comment]
I don't see what the problem is.  The test library does
allow_redirects=True .

The output suggests that `requests` is retrying with a GET request when the
original request was a POST. According to spec it should retry with a
second POST if the status was 301, 302, or 307. Using a GET after a POST
redirect would only be appropriate for 303.

Hmm... https://tools.ietf.org/html/rfc7231#section-6.4.2

Note: For historical reasons, a user agent MAY change the request
      method from POST to GET for the subsequent request.  If this
      behavior is undesired, the 307 (Temporary Redirect) status code
      can be used instead.

UCK.  This an early spec violation on Microsoft's part, and the recently
revised HTTP spec has turned it from a "you have to do it right" to "it's
ok if you do it wrong". I bet this is what's happening...​

File: ws-tests/run_tests.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    exit 1
+fi
+num_t=0
+num_p=0
+failed=''
+for fn in $(ls test_*.py)
+do
+    if test $fn = skipped_test_name_here.py ; 
+    then
+        echo test $fn skipped
```
Comment (jimallman) [inline ws-tests/run_tests.sh]: Should this be ```/bin/echo -n "s"```, for the behavior described below?

**Comment by kcranston** [comment]
Merging, noting that the discussion here doesn't have much to do with this PR.

**Pull request merged**